gaeguli/fifo-transmit.h

```diff
@@ -58,6 +61,11 @@ gboolean                gaeguli_fifo_transmit_stop     (GaeguliFifoTransmit    *
                                                         guint                   transmit_id,
                                                         GError                **error); 
 
+typedef enum {
+  GAEGULI_FIFO_TRANSMIT_ERROR_FAILED,
+  GAEGULI_FIFO_TRANSMIT_ERROR_ADDRINUSE
+} GstPlayerError;
```

Comment:
this name is mistakenly used? 
But, I think it's better to have a more general term for this type of errors because actually we can have a different type of transmitter later.
How about `GaeguliTransmitError` in *type.h*?

Reply:
Of course, the name is a mistake.
I'll also move the enumeration.